### ISSUE TITLE
LocalStorageCache filter doesn't handle QuotaExceededError

### ISSUE THREAD

**Issue by deweydb** [apache/incubator-pagespeed-mod#1275]
I'm getting quota exceeded errors client side while running ngx_pagespeed module.
`
Uncaught QuotaExceededError: Failed to execute 'setItem' on 'Storage': Setting the value of 'pagespeed_lsc_url:https://bid13.com/sites/default/files/imagecache/product_list/19-1_0.jpg pagespeed_lsc_hash:FceD9qKAq8' exceeded the quota.`

This error occurs in chrome (Version 48.0.2564.116 (64-bit)  on this page: https://bid13.com

Please let me know if i can provide any further information.

**Comment by jeffkaufman** [comment]
It sounds to me like you enabled the local storage caching filter [1] and you have enough images that you hit Chrome's quota?

This isn't a problem by itself, but you could consider turning off the filter.

[1] https://developers.google.com/speed/pagespeed/module/filter-local-storage-cache

**Comment by deweydb** [comment]
Yeah i disabled the filter for now. it would be nice if i could leave it enabled and there was some sort of check for amount of storage space so errors like this didn't occur.

**Comment by jeffkaufman** [comment]
It sounds like at the least we should be catching this error.
